tests/execution_engine/test_sqlalchemy_batch_data.py

```diff
@@ -166,7 +166,7 @@ def test_instantiation_with_temp_table_schema():
         create_temp_table=True,
         temp_table_schema_name="test_schema",
     )
-    assert "test_schema" not in engine.execute.call_args[0][0]
+    assert "test_schema" not in str(engine.execute.call_args[0][0])
```

Comment:
This is the test that needed to be changed. The `call_args[0][0]` now returns a `TextClause`, which is converted to a `str()`